contracts-periphery/test/UniswapWithdrawHook.fork.1.t.sol

```diff
@@ -118,7 +131,7 @@ contract UniswapWithdrawHookTest is DeployUmbraTest {
     amount = bound(amount, 10e10, 100_000 ether);
     swapAmount = bound(swapAmount, 10e8, amount);
     feeBips = bound(feeBips, 1, 100);
-    sponsorFee = bound(sponsorFee, 1, amount - swapAmount);
+    sponsorFee = bound(sponsorFee, 0, amount - swapAmount);
```

Comment:
Was getting this so I changed the min
[FAIL. Reason: StdUtils bound(uint256,uint256,uint256): Max is less than min. Counterexample: calldata=0x3d93c91a0000000000000000000000000000000000000000000000000000000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, args=[0, 115792089237316195423570985008687907853269984665640564039457584007913129639935, 0, 0]] testFuzz_HookTest_withdrawTokenAndCall(uint256,uint256,uint256,uint256) (runs: 296, μ: 2925804, ~: 2905346)